app/src/main/java/org/schabi/newpipe/fragments/MainFragment.java

```diff
@@ -112,6 +148,14 @@ public void onTabReselected(TabLayout.Tab tab) {
         @Override
         public Fragment getItem(int position) {
             switch (position) {
+                case 0:
+                    if(PreferenceManager.getDefaultSharedPreferences(getActivity())
```

Comment:
Why not just remove subscription page as a kiosk candidate? This kind of check might be problematic when there are more than 2 pagers on the main fragment.

Reply:
Well fix that once more pages are there. Also some people requested the Subscriptionpage to be shown directly when NewPipe is opened.